hemkan/gourmet-guide
# Download Button + Clear Button + Formatting

## Description
This PR introduces new "Download" and "Clear" buttons to enhance functionality and improves overall formatting across the application.

**Related Issues:**
Resolves #3.

### Changes
`app/api/chat/route.js`: 
- Adjusted system prompt for 300 words 
- Increased token size for recipe responses

`app/page.js`: 
- Fixed scroll-to-bottom
- Added download chat feature
- Corrected formatting for cropped images
- Updated chat component colors
- Added Markdown formatting for chat output to improve recipes formatting

`public/IceCreamDoodle.svg`: Cropped SVG to fit the page

### Testing
1. Install Dependencies
```
npm i
```
2. Start the development server
```
npm run dev
```
3. Configure `.env.local`
```
GROQ_API_KEY=your_api_key
```
Once the application is successfully running at http://localhost:3000, log in to access the chat interface, where you can interact with the features and verify changes.

### Relevant Screenshots
<img src="https://github.com/user-attachments/assets/ae89b328-030a-4d52-a911-458cdadfcdea" width="300"> <img src="https://github.com/user-attachments/assets/1a545daa-d31f-4cfc-b0e1-823a4254bde3" width="300"> 
<img src="https://github.com/user-attachments/assets/e7c8c37b-1819-4e7d-b423-0e15fba30c06" height="300"> <img src="https://github.com/user-attachments/assets/3c44dbe3-aa17-42b6-9b13-f015d84043dc" height="300">
